test/addons/olm/crds/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+---
+resources:
+  - https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.22.0/crds.yaml
```

Comment:
This belongs to the static kustomization commit.

It is nice that you added crds directory for this kustomization - but I we do install crds as part of the same kustomization (e.g. rook-operator), so I wonder if we can simplify by having one kustomization, referring to all resources.

This requires testing - if we do have to deploy the crds first and wait until the crds are established, we must keep 2 kustomizations and caches. In this case we want to have consistent directory layout like:

    addons/
      olm/
        crds/
          kustomization.yaml
        operator/
          kustomization.yaml
        start
        fetch